frontend/packages/core/src/accordion.tsx

```diff
@@ -159,8 +159,13 @@ export const AccordionGroup = ({ children, defaultExpandedIdx }: AccordionGroupP
 };
 
 const StyledAccordionDetails = styled(MuiAccordionDetails)({
+  display: "flex",
+  flexWrap: "wrap",
+  boxSizing: "border-box",
   "> *": {
     padding: "8px 8px",
+    flex: "1 50%",
+
```

Comment:
dont these belong in the resolver or form somehow? i don't think applying them to all accordiondetails makes sense

Reply:
Good point. I think probably moving to the resolver makes more sense.